eagleEvents/models/event.py

```diff
@@ -10,11 +10,11 @@
 class Event(db.Model):
     __tablename__ = 'event'
     id = db.Column(UUIDType(binary=False), primary_key=True, default=uuid.uuid4)
-    time = db.Column(db.DateTime, nullable=False)
+    time = db.Column(db.DateTime, nullable=False, default=datetime.datetime.now())
```

Comment:
Does it really make sense to default an event time to right now?  I didn't put a default in originally because I couldn't decide on a good one.

Reply:
I left that in there by accident while I was trying to work on date formatting between python and html. I have taken it out now.